language-support/ts/daml-ledger/index.ts

```diff
@@ -493,10 +493,10 @@ class Ledger {
     let haveSeenEvents = false;
     let state = init;
     const emitter = new EventEmitter();
-    ws.onopen = () => {
+    ws.addEventListener('open', () => {
```

Comment:
why did you change these?

Reply:
That makes mocking easier since overriding a method is easier than overriding a field with magic behavior.